src/main/resources/assets/jeresources/lang/it_it.json

```diff
@@ -0,0 +1,167 @@
+{
+	"jer.config.itemsPerColumn.title": "Oggetti per colonna",
+	"jer.config.itemsPerColumn.description": "Cambia la quantità di oggetti per colonna nella scheda Dungeon e Creature della JEI",
+	"jer.config.itemsPerRow.title": "Oggetti per riga",
+	"jer.config.itemsPerRow.description": "Cambia il numero gi oggetti per riga nella scheda Dungeon della JEI",
+	"jer.config.enchantsBlacklist.title": "Lista Nera degli incantesimi",
+	"jer.config.enchantsBlacklist.description": "Rimuove gli incantesimi dalla scheda Incantesimi della JEI",
+	"jer.config.diyData.title": "Usa i dati DIY nelle schede",
+	"jer.config.diyData.description": "The build in compat code will not load and jsons will be loaded (requires MC restart)",
```

Comment:
The line is used a comment/description in the config window.  
The build in compat code (standard JER infromation) will not load and jsons (DIY version of ore graphs) will be loaded (requires MC restart)